src/Extension/UrlExtension.php

```diff
@@ -48,6 +50,22 @@ public function register(Engine $engine) : void
     {
         $engine->registerFunction('url', [$this, 'generateUrl']);
         $engine->registerFunction('serverurl', [$this, 'generateServerUrl']);
+        $engine->registerFunction('route', [$this, 'getRouteResult']);
+    }
+
+    /**
+     * Get the RouteResult instance of UrlHelper
+     *
+     * @return RouteResult
+     */
+    public function getRouteResult() : RouteResult
+    {
+        if (! method_exists($this->urlHelper, 'getRouteResult')) {
+            throw new Exception\RuntimeException(
+                'The helper does not contains the route result. Please update zend-expressive-helpers'
+            );
+        }
```

Comment:
do we need that exception? Maybe we should bump minimum version of `zend-expressive-helpers` once released?

If needed, we should add `@throws` annotation on the method.

Reply:
I've bumped the minimum supported zend-expressive-helpers version to 5.2 at this time, which allows removing the new exception, as well as the check here.